spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -292,12 +271,18 @@ object SpdxDeclaredLicenseMapping {
         "http://www.apache.org/licenses/LICENSE-2.0.txt" to APACHE_2_0.toExpression(),
         "http://www.gnu.org/copyleft/lesser.html" to LGPL_3_0_ONLY.toExpression(),
         "https://raw.github.com/RDFLib/rdflib/master/LICENSE" to BSD_3_CLAUSE.toExpression(),
-        "new BSD" to BSD_3_CLAUSE.toExpression(),
         "public domain, Python, 2-Clause BSD, GPL 3 (see COPYING.txt)"
                 to (((licenseRef("public-domain-disclaimer", "scancode") and PYTHON_2_0.toExpression())
                 and BSD_2_CLAUSE.toExpression()) and GPL_3_0_ONLY.toExpression()),
         "the Apache License, ASL Version 2.0" to APACHE_2_0.toExpression()
-    )
+    ).let { caseSensitiveMap ->
+        caseSensitiveMap.toSortedMap(String.CASE_INSENSITIVE_ORDER).also { caseInsensitiveMap ->
+            val difference = caseSensitiveMap.keys.subtract(caseInsensitiveMap.keys)
+            require(difference.isEmpty()) {
+                "The following ${difference.size} keys are present in different capitalizations: $difference"
+            }
```

Comment:
You could put the four lines above inside a `if (caseInsensitiveMap.size > caseSensitiveMap.size)` as a minor performance optimization (also in the class below).

Reply:
Done, but the other way around: `caseSensitiveMap.size > caseInsensitiveMap.size`.